DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/RevitConverterToSpeckle.cs

```diff
@@ -29,24 +28,43 @@ ParameterValueExtractor parameterValueExtractor
   // if it cannot be converted then we should throw
   public Base Convert(object target)
   {
-    var objectConverter = _toSpeckle.ResolveInstance(target.GetType().Name);
+    var objectConverter = GetConversionForObject(target.GetType());
 
     if (objectConverter == null)
     {
-      throw new NotSupportedException($"No conversion found for {target.GetType().Name}");
+      throw new SpeckleConversionException($"No conversion found for {target.GetType().Name}");
```

Comment:
Why not `NotSupportedException`? In my mind, `SpeckleConversionException` is something that would happen during conversion, as opposed to not having a conversion at all.

Being able to tell the two apart is quite important, as not finding a converter for a given object is not an "error" bur rather should be treated as a warning to the user (when we get better UI for it)

Reply:
I think the advantage is minimizing the different exceptions that the converter can throw. We probably need to just make more specific SpeckleExceptions